tests/dummy/app/pods/docs/testing/acceptance-tests/template.md

```diff
@@ -52,9 +52,9 @@ Say we wanted to test that when the user visits a details route for a movie titl
 
 ```js
 // mirage/factories/movie.js
-import Mirage from 'ember-cli-mirage';
+import Factory from 'ember-cli-mirage';
```

Comment:
@cah-briangantzler should this be `import { Factory } from 'miragejs';`?

Reply:
Yep. Fixed it. Though IMO Most the of the Docs like this need removed. Ember-cli-mirage is no longer of the source of Factories. This same documentation exists in MirageJS. About 80% of the current Docs should be removed.